github/pom.xml

```diff
@@ -95,6 +95,27 @@
             <groupId>org.hamcrest</groupId>
             <artifactId>hamcrest-junit</artifactId>
         </dependency>
+
+        <!-- Arquillian recorder reporter -->
+        <dependency>
+            <groupId>org.arquillian.extension</groupId>
+            <artifactId>arquillian-recorder-reporter-spi</artifactId>
+            <version>${version.arquillian.recorder}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.arquillian.extension</groupId>
+            <artifactId>arquillian-recorder-reporter-api</artifactId>
+            <version>${version.arquillian.recorder}</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <!--AssertJ for assertion -->
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>${version.assertj}</version>
```

Comment:
@dipak-pawar no, if dependencyManagement sets an scope you don't need to set again